pkgs/applications/misc/sandbar/default.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pixman,
+  fcft,
+  pkg-config,
+  wayland-protocols,
+  wayland-scanner,
+  wayland
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "sandbar";
+  version = "v0.1";
```

```suggestion
  version = "0.1";
```

See https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming

Please also update commit message.